docs/topics/accessing-unmatched-rules.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="accessing-unmatched-rules_{context}"]
+= Accessing unmatched rules
+
+To access unmatched rules, you must run the analysis with enhanced logging enabled.
+
+. Navigate to *Advanced* under *Application analysis*.
+. Select *Options*.
+. Check *Enhanced advanced analysis details*.
+
+When you run an analysis:
+
+. Navigate to *Reports* in the side drawer.
+. Click *View analysis details*, which opens the YAML/JSON format log view.
+. Select the `issues.yaml` file.
+. For each ruleset, there is an *unmatched* section that lists the rule IDs that do not find match rules.
```

Comment:
@kpunwatk - this is what was changed? How can it be improved?
Thanks

Reply:
Thanks, @anarnold97, the changes look perfect to me. Just a note: since issues.yaml is a YAML file, it will display in a YAML/Text format log view.